src/ui/Folding/Folding.ts

```diff
@@ -144,6 +144,8 @@ export class Folding extends Component {
      * `@myfield descending`, etc. [See
      * Query Parameters - sortCriteria](https://developers.coveo.com/x/iwEv#QueryParameters-sortCriteriasortCriteria)).
      *
+     * TODO : Explain technical limitation related to client side sorting.
```

Comment:
@amoreauCoveo Ping me when you are ready. It's a bit complicated to explain through code review, it will be better to explain in person.

Reply:
Preferably I'd like if this PR did not stay open for weeks. 

If you do not have enough time, me do write gooder words no eror.